search-app/src/cmr/search/services/query_execution/facets/facets_v2_results_feature.clj

```diff
@@ -5,13 +5,14 @@
    [camel-snake-kebab.core :as csk]
    [clojure.set :as set]
    [clojure.string :as string]
-   [cmr.elastic-utils.search.query-execution :as query-execution]
+   [cmr.common.log :refer [debug info]]
```

Comment:
Remove as it isn't needed. I can't find any debug or info log statements in this namespace.